modules/battery_manager/README.md

```diff
@@ -1,5 +1,76 @@
 # Battery Manager
 
+### LEVEL easy:
+ - Subproject: 
+   - If the test battery entry does NOT have a `subprojectID` 
+   (`subprojectID=NULL`), the instrument gets administered to ALL subprojects.
+   - If the test battery entry has a `subprojectID` set and the `subprojectID`
+   matches the one of the timepoint, the instrument is administered only to 
+   that subproject.
+
+ - Stage: 
+   - If the test battery entry does NOT have a `stage` (`stage=NULL`), the 
+   instrument is NEVER administered.
+   - If the test battery entry has a `stage` set and the `stage` matches the 
+   one of the timepoint, the instrument is administered at that stage.
+
+ - Center:
+   - If the test battery entry does NOT have a `CenterID` (`CenterID=NULL`), 
+   the instrument gets administered at ALL centers.
+   - If the test battery entry has a `CenterID` set and the `CenterID` matches
+   the one of the timepoint, the instrument is administered at that CenterID.
+
+### LEVEL medium:
+ - AgeMinDays/AgeMaxDays:
+   - If the test battery entry does NOT have a `AgeMinDays` or a `AgeMaxDays` 
+   or both (`AgeMinDays=NULL` or `AgeMaxDays=NULL`), the instrument is NEVER 
+   administered.
+   - If the test battery entry has `AgeMinDays` and `AgeMaxDays` set and they 
+   are both set to `0`, the instrument gets administered at ALL ages;
+   - If the test battery entry has `AgeMinDays` and `AgeMaxDays` set and they 
+   are set to any value other than `0`, the instrument gets administered IF AND
+   ONLY IF the age of the candidate at the timepoint is between `AgeMinDays` 
+   and `AgeMaxDays`;
+
+### LEVEL hard:
+
+ - VisitLabel:
+   - If the test battery entry does NOT have a `Visit_label`
+   (`Visit_label=NULL`), the instrument gets administered IF AND ONLY IF no
+   other test battery entries matches the timepoint's visit label and
+   subproject.
+   - If the test battery entry has a `Visit_label` set and the `Visit_label`
+   matches the one of the timepoint, the instrument is administered at that
+   Visit.
+   - ***NOTE:** In order to administer an instrument at all visits without
```

Comment:
It looks like the bold got messed up here

Reply:
euhm the intent was that the note is in bold and italics and the rest just in italics...

Now its **NOTE** in bold and the *rest in italics*.

I dont care wither way just FYI